ios/RNCSafeAreaViewEdgeModes.h

```diff
@@ -0,0 +1,11 @@
+#import <Foundation/Foundation.h>
+
+typedef NS_ENUM(NSInteger, RNCSafeAreaViewEdgeModes) {
+    RNCSafeAreaViewEdgeModesAdditive = 0b1000,
```

Comment:
It might be better to leave these as the defaults since you can't use bitwise operations on this type. Also, if you put `off` first, it'll have a value 0